local-test-infra/local-partner-deployment.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: partner
```

Comment:
Now that it will be in the default namespace, wouldn't it be good to add a "tnf" prefix to the name of this pod? Something like "tnf-partner" / "tnf_partner".